modules/sizing-requirements-for-gitops.adoc

```diff
@@ -0,0 +1,28 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/about-sizing-requirements-gitops.adoc
+
+:_content-type: CONCEPT
+[id="sizing-requirements-for-gitops_{context}"]
+= Sizing requirements for GitOps
+
+Red Hat Openshift GitOps is a declarative way to implement continuous deployment for cloud native applications. Through GitOps, you can define and configure the CPU and Memory requirements of your application. Everytime you install the GitOps operator, the resources on the namespace is installed within the defined limits. If the default install does not set any limits or requests, the operator will fail within the namespace which has quotas. Without enough resources, the cluster will not be able to schedule ArgoCD related pods. The resource requests and limits for the default workloads are:
+
+[cols=",,,,",options="header",]
+|===
+|Workload |CPU requests |CPU limits |Mem requests |Mem limits
+|argocd-application-controller |1 |2 |1024M |2048M
+|applicationset-controller |1 |2 |512M |1024M
+|argocd-server |0.125 |0.5 |128M |256M
+|argocd-repo-server |0.5 |1 |256M |1024M
+|argocd-redis |0.25 |0.5 |128M |256M
+|argocd-dex |0.25 |0.5 |128M |256M
+|HAProxy |0.25 |0.5 |128M |256M
+|===
+
+Optionally, you can also use the ArgoCD custom resource with `kubectl`/`oc` command to see the specifics and modify them:
```

Comment:
```suggestion
Optionally, you can also use the ArgoCD custom resource with the `kubectl` or `oc` command to see the specifics and modify them:
```

**Question:** When using OCP we always replace kubectl with oc, right? Please check and retain only oc if possible. Refer to the OCP documentation and do the needful as appropriate.